src/vendor/ajv.ts

```diff
@@ -0,0 +1,62 @@
+// Forked from https://github.com/eslint/eslint/blob/7fe4114be2e714506fd406ea4474430ea3de0f93/lib/config/flat-config-helpers.js and https://github.com/eslint/eslint/blob/7fe4114be2e714506fd406ea4474430ea3de0f93/lib/shared/ajv.js
+
+import Ajv from 'ajv'
+import metaSchema from 'ajv/lib/refs/json-schema-draft-04.json'
```

Comment:
When I try to run tests with this, I get the following:
```
TypeError: Module "file:///path/to/node_modules/ajv/lib/refs/json-schema-draft-04.json" needs an import attribute of "type: json"
```

I'll try to put together a minimal reproducible case, but I'm curious if anybody else has worked around the same.

Reply:
In the meantime, downgrading to `eslint-vitest-rule-tester@v0.6.1` solves the problem.